python/ray/data/_internal/execution/interfaces/common.py

```diff
@@ -0,0 +1,2 @@
+# Node id string returned by `ray.get_runtime_context().get_node_id()`.
```

Comment:
would it be better if we call this file as `types.py`, so it only stores types?

Reply:
I thought about that before. But later I thought using type alias is only an implementation detail that may change in the future. I think type aliases are good for simple use cases. but when the use cases become more complex, they make the code less readable. And we'll want to convert them to classes.  Then I prefer not using file names like "types.py".

For this particular `NodeIdStr`, my preference is "define in ray core" > "define in a more common "common" folder for the whole data lib" > "define here". But to avoid this PR changing too many things, I'd prefer the current way for now.